rtgui/edit.h

```diff
@@ -152,6 +145,59 @@ class EditBuffer;
  *
  */
 
+
+
+class ObjectMOBuffer
+{
+private:
+
+    // Used to draw the objects where the color correspond to the object's ID, in order to find the correct object when hovering
```

Comment:
I'll use FORMAT_RGB16_565 in the next commit then.

For the general purpose scene graph library, I could have used one if I knew their existence at the time of adding this feature, but the classes already exist in RT now, we're not bound to a third party, and it has the extra possibility to define a different associated "mouse over" geometry than the displayed one (e.g. the central "mouse over" spot of the Gradient widget is far bigger than the displayed one).